src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,15 @@
 
 Release History
 ===============
+2.38.2
+++++++
+
+This version is only released for CentOS 7 and RHEL 7.
+
+**App Service**
+
+* Backport Breaking Change: Redact appsettings, storage-account, configs (#28671)
+* Backport Breaking Change: Redact keys (#28751)
```

Comment:
The history notes are not very clear. We should clearly convey what has been changed.

Reply:
I've copied the history note from 2.53.1 and 2.54.0